mesh/mesh.cpp

```diff
@@ -10469,13 +10507,61 @@ GeometricFactors::GeometricFactors(const Mesh *mesh, const IntegrationRule &ir,
       eval_flags |= QuadratureInterpolator::DETERMINANTS;
    }
 
-   const QuadratureInterpolator *qi = fespace->GetQuadratureInterpolator(ir);
+   const QuadratureInterpolator *qi = fespace->GetQuadratureInterpolator(*IntRule);
+   // For now, we are not using tensor product evaluation (not implemented)
```

Comment:
This seems to be a copy/paste - if so, is it factorize-able?

Reply:
That is a good point, done!